examples/sample_streaming_measurement/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "sample_streaming_measurement"
+version = "0.1.0"
+description = "MeasurementLink example that generates a predictable stream of data."
+authors = ["National Instruments"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+ni-measurementlink-service = {version = "^1.1.0-dev0", allow-prereleases = true}
+click = ">=7.1.2"
+
+[tool.poetry.group.dev.dependencies]
+mypy = ">=1.0"
+# Uncomment to use prerelease dependencies (requires poetry>=1.2).
+# ni-measurementlink-service = {path = "../..", develop = true}
+
+[build-system]
+requires = ["poetry-core>=1.2.0"]
+build-backend = "poetry.core.masonry.api"
+
+[[tool.mypy.overrides]]
+module = [
+    "grpc.*",
+]
+ignore_missing_imports = true
```

Comment:
Incorporate the changes from https://github.com/ni/measurementlink-python/pull/275 
```suggestion
[tool.poetry.group.dev.dependencies]
mypy = ">=1.0"
grpc-stubs = "^1.49.1"
# Uncomment to use prerelease dependencies (requires poetry>=1.2).
# ni-measurementlink-service = {path = "../..", develop = true}

[build-system]
requires = ["poetry-core>=1.2.0"]
build-backend = "poetry.core.masonry.api"
```